jsfh.yml

```diff
@@ -0,0 +1,4 @@
+expand_buttons: true
+collapse_long_descriptions: false
+link_to_reused_ref: false
+examples_as_yaml: true
```

Comment:
Looks like that option was very recently added. I now wanted to use it, but couldn't get it to actually work: https://github.com/coveooss/json-schema-for-humans/pull/144#issuecomment-1211895019.

Reply:
It seems this option really is not part of their latest release (0.41.6), so let's leave it as-is for now and add it later once available.

Reply:
Since merging this PR got delayed anyway, consider adding this now. I have confirmed that the option is working in the latest release of `json-schema-for humans` (0.41.8).

Reply:
Thanks for pointing that out! Added that now.